### PR TITLE
Task/tam 6/pet death

### PR DESCRIPTION
**Pull request by kalibria**

### CODE REVIEW

File: src/App.module.css [hunk 1]
```diff
@@ -7,13 +7,50 @@
...
+    top: 50%;
+    width: 250px;
+    height: auto;
+    text-align: center;
+    padding: 15px;
+}
+
+.wrapperCloseButton {
+position: relative;
+}
```
Comment (ikushlianski) [inline src/App.module.css]: pls try to adhere to consistent formatting (indents)

File: src/name/EnterName.js [hunk 2]
```diff
@@ -1,36 +1,41 @@
...
-
-    const handleChange = (e) => {
-        setLetters(e.target.value);
+const EnterName = ({ name, setPetName }) => {
+  const [letters, setLetters] = useState(name);
+  const [error, setError] = useState(false);
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    if (isValid(letters) && name !== " ") {
```
Comment (ikushlianski) [inline src/name/EnterName.js:12]: you are checking only for one space, but what if I enter multiple, like `" multiple spaces   "`?

**Review by ikushlianski — APPROVED** [review]

File: src/App.module.css (hunk 1 above)
Comment (kalibria) [reply src/App.module.css]: fixed

File: src/name/EnterName.js (hunk 2 above)
Comment (kalibria) [reply src/name/EnterName.js:12]: the program will trim multiple spaces to one

**Pull request merged**